src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let check = 0;
+
+  return (number1) => {
+    return (number2) => {
+      check++;
+
+      if (check > 3 & check % 2 === 0) {
```

Comment:
I see what you tried there :D

I think it'd be better to use &&

Still will approve on my end, I suggest you change it

Reply:
agree with @UmizDemud here)